pkg/clients/resourcerecordset/resourcerecordset.go

```diff
@@ -44,84 +43,92 @@ type Client interface {
 	ListResourceRecordSetsRequest(input *route53.ListResourceRecordSetsInput) route53.ListResourceRecordSetsRequest
 }
 
-// RRsetNotFound will be raised when there is no ResourceRecordSet
-type RRsetNotFound struct{}
+// NotFoundError will be raised when there is no ResourceRecordSet
+type NotFoundError struct{}
```

Comment:
Yeah, there is no not found error for resource record sets because they only have `LIST` method, which would just return 0 elements if nothing was found. The workaround implemented here is to return a handmade error in case there is no `ResourceRecordSet` in the list that matches our identifier set.